src/Util/ErrorFormatGuesser.php

```diff
@@ -37,14 +37,24 @@ private function __construct()
     public static function guessErrorFormat(Request $request, array $errorFormats): array
     {
         $requestFormat = $request->getRequestFormat('');
+
         if ('' !== $requestFormat && isset($errorFormats[$requestFormat])) {
             return ['key' => $requestFormat, 'value' => $errorFormats[$requestFormat]];
         }
 
-        foreach ($errorFormats as $key => $value) {
-            return ['key' => $key, 'value' => $value];
+        $requestMimeTypes = Request::getMimeTypes($request->getRequestFormat());
+        $defaultFormat = [];
+
+        foreach ($errorFormats as $format => $errorMimeTypes) {
+            if ([] === $defaultFormat) {
```

Comment:
what? this looks weird can we change to empty? or `!$defaultFormat`? Why can't we init above with the proper array?

Reply:
Because we not sure the key 0 exist, it looks ok to me to do this. We indeed may use `!$defaultFormat`.

Reply:
I'd be :+1: for `!$defaultFormat`. Anyway I panicked because `[] === []` is false in javascript :laughing:

Reply:
Fixed.